trpc/router/project.ts

```diff
@@ -422,6 +422,29 @@ export const project = router({
                 })
                 .then((res) => res.map((project) => addCalculatedProjectFields(project, activityId, role)));
         }),
+    getStarred: protectedProcedure.query(async ({ ctx }) => {
+        const { activityId, role } = ctx.session.user;
+
+        const whereQuery = {
+            stargizers: {
+                some: {
+                    id: activityId,
+                },
+            },
+        };
+        return prisma.project
```

Comment:
Please, rewrite this query with kysely instead of prisma. If you have any questions about of sql and kysely then you can ask me